deployment/stand-alone.md

```diff
@@ -138,7 +142,9 @@ server {
 }
 ```
 
-Then, symlink these to sites-enabled: `sudo ln -s /etc/nginx/sites-available/public /etc/nginx/sites-enabled && sudo ln -s /etc/nginx/sites-available/apply /etc/nginx/sites-enabled`. Then restart nginx using `sudo systemctl restart nginx`.
+The `client_max_body_size` configuration directive is very important. Hypha uploads files to the server in 2.5MB chunks. However, by default, nginx limits uploads to 1MB chunks. The result is that users of a Hypha system running "behind" nginx will see file upload failures for any file larger than 1MB. Using the `client_max_body_size` directive in the nginx `server` context is, therefore, required to make it possible for users to upload files that are bigger than 1MB in size.
+
+Symbolically link these to sites-enabled: `sudo ln -s /etc/nginx/sites-available/public /etc/nginx/sites-enabled && sudo ln -s /etc/nginx/sites-available/apply /etc/nginx/sites-enabled`. Then restart nginx using `sudo systemctl restart nginx`.
```

Comment:
@hawkinsw Curious why you changed the symlink language here?  I don't think it should be a merge blocker, and I don't actually have any strong opinion as to whether the original symlink language was better or the new language is better, it's just that it's unrelated to the `client_max_body_size` change.

Again, I don't think it should block merging of the change.  I'm just pointing out the presence of an unrelated change because it's my job to be a nudge :-).